### ISSUE TITLE
Supporting multiple instances of WS servers

### ISSUE THREAD

**Issue by iamsaso** [graph-gophers/graphql-transport-ws#8]
The current implementation does not support running WS servers on multiple instances and the client will not receive the message if it's subscribed on a different instance.

A solution would be to implement Redis as a fanout backend to redistribute messages on other instances.

**Comment by pavelnikolov** [comment]
That is correct and in order to achieve this some kind of pub/sub system is required. PRs are more than welcome, though.
